infor-design/enterprise
# 5865/6331/6348 - Add and fix searchfield toolbar new designs

**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR enhances the searchfield toolbar components for better UI and accessibility. The design was provided here https://www.figma.com/file/gS5jIE14am8hYEvmtIAXLv/IDS-Search?node-id=209%3A2536 for design references.

**Related github/jira issue (required)**:

Closes #5865 
Closes #6331 
Closes #6348

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app

1.) Toolbar Flex
- Go to http://localhost:4000/components/toolbar-flex/example-index.html
- The toolbar should have
  -  a white background
  - icons themselves should have 18x18 sizes (search and close icon)
  - when active, it should have an azure border-color
  - for more detailed information, go to the Figma file presented above
  - test to all modes (light, dark, contrast)

2.) Header
- Go to http://localhost:4000/components/header/example-searchfield-expanded.html
- Placeholder should be visible
- Height should be 38px on both new and classic
- Type anything in the field, it should show the close icon
- Hover the close icon, it should not disappear
- test to all modes (light, dark, contrast)

3.) Tabs Module
- Go to http://localhost:4000/components/tabs-module/example-category-searchfield-go-button-home.html?theme=new&mode=light&colors=2578a9
- Category Icon should be visible
- Close icon should also be visible
- All elements should be 38px in height
- Test on classic too

4.) Lookup
- Go to http://localhost:4000/components/lookup/example-index
- Open the lookup
- The searchfield should have a new design 

**Included in this Pull Request**:
~~- [ ] An e2e or functional test for the bug or feature.~~
